services/common/src/components/FormWrapper.tsx

```diff
@@ -0,0 +1,93 @@
+import { Button, Row, Typography } from "antd";
+import React from "react";
+
+export const { Provider: FormProvider, Consumer: FormConsumer } = React.createContext({
+  view: false,
+});
```

Comment:
the Provider/Consumer is my favourite part! Before I put it in it was pretty annoying to have to add a parameter to each input.